source/declarations.tex

```diff
@@ -735,6 +735,23 @@
 typedef decltype([]{}) C;       // the closure type has no name for linkage purposes
 \end{codeblock}
 \end{example}
+An unnamed class with a typedef name for linkage purposes shall not
+\begin{itemize}
+\item
+  declare any members
+  other than non-static data members, member enumerations, or member classes,
+\item
+  have any base classes or default member initializers, or
+  contain a lambda-expression,
```

Comment:
Missing `\item` before "contain a lambda-expression" and, lambda-expression should be \grammarterm'ed.

Reply:
Fixed.